### DISCUSSION TITLE
Plotting several pole figures in one figure

### DISCUSSION THREAD

**Discussion by mfuhr-mpg** [mtex-toolbox/mtex#1576] [Ask Anything] [answered]
Hi,
I have orientation information (from EBSD) of different samples that I want to evaluate as (110) pole figure in one figure, i.e. I want to compare the samples based on their pole figure. Using MATLAB´s `subplot `doesn´t really work for me.
Does anyone have an elegant solution using the MTEX tools?
Thanks in advance!
Max

**Comment by ralfHielscher** [comment, marked as answer]
Try
```matlab
plotPDF(ori1,Miller(1,1,0,CS))
nextAxis
plotPDF(ori2,Miller(1,1,0,CS))
nextAxis
plotPDF(ori3,Miller(1,1,0,CS))
```
Ralf.

**Reply by mfuhr-mpg** [reply]
Perfect, works like a charm! Thanks!
